ulib/c_libax/src/unistd.c

```diff
@@ -6,6 +6,8 @@
 #include <time.h>
 #include <unistd.h>
 
+char **__environ = 0;
```

Comment:
Move to `env.c`

Reply:
Move to `env.c` and rename to `char **environ`.
TODO: This should be initialized some where in libc?